src/programs/ectrans.in

```diff
@@ -50,7 +50,7 @@ info()
   echo "    flags         : @EC_Fortran_FLAGS@"
   echo ""
   echo "Features:"
-  echo "  MPI             : @ectrans_HAVE_MPI@"
```

Comment:
`ectrans_HAVE_MPI` is still available. Please revert.

Reply:
Are you sure? This is taken from redgreengpu: https://github.com/ecmwf-ifs/ectrans/blob/redgreengpu/src/programs/ectrans.in#L53

Reply:
I guess it makes more sense to refer to the ecTrans variable. The MPI feature depends on `fiat_HAVE_MPI` anyway.